src/transformation/utils/scope.ts

```diff
@@ -179,69 +216,73 @@ function shouldHoistSymbol(context: TransformationContext, symbolId: lua.SymbolI
     return false;
 }
 
-function hoistVariableDeclarations(
-    context: TransformationContext,
-    scope: Scope,
-    statements: lua.Statement[]
-): lua.Statement[] {
+function hoistVariableDeclarations(context: TransformationContext, scope: Scope, statements: lua.Statement[]) {
```

Comment:
Think I would prefer if you explicitly declared the return type of these hoist* functions for clarity - I guess it's just HoistingResult?